src/BasePlatform.ts

```diff
@@ -72,7 +72,7 @@ export default abstract class BasePlatform {
     protected _favicon?: Favicon;
 
     protected constructor() {
-        dis.register(this.onAction);
+        dis.register(this.onAction.bind(this));
```

Comment:
Why is this bit necessary?

Reply:
You can't extend methods which aren't methods, arrow-functions are class fields which are just variables written during the class construction implicitly so cannot be extended, only overwritten (no access to `super.<method>`)